### ISSUE TITLE
panic at index out of bounds

### ISSUE THREAD

**Issue by matthiasbeyer** [lusingander/serie#9]
```
thread 'main' panicked at src/graph/calc.rs:228:5:                                                                                                                                                 
index out of bounds: the len is 5 but the index is 5                                                                                                                                               
stack backtrace:                                                                                                                                                                                   
   0: rust_begin_unwind                                                                                                                                                                            
   1: core::panicking::panic_fmt                                                                                                                                                                   
   2: core::panicking::panic_bounds_check                                                                                                                                                          
   3: serie::graph::calc::calc_graph                                                                                                                                                               
   4: serie::run                                                                                                                                                                                   
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

**Comment by lusingander** [comment]
What repository are you running? I don't know the details so there's nothing I can do.

**Comment by matthiasbeyer** [comment]
I am running 0.1.0

**Comment by lusingander** [comment]
Probably the underlying problem is the same as #2.

**Comment by juliolinarez** [comment]
Same error.  (serie v0.1.0) (installed with brew). Private repository

```
thread 'main' panicked at src/graph/calc.rs:228:5:
index out of bounds: the len is 47 but the index is 47
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by lusingander** [comment]
I'm not entirely sure what your specific problem is, but #5 has been merged so that might solve your problem. It hasn't been released yet so please build it if you need it.

**Comment by alerque** [comment]
Same problem here, but only on some repos.

**Comment by AgustinBerardi** [comment]
I had the same issue before the last release (version 0.1.0). With the new update (v0.1.1), here is the new error:
![Uploading Screenshot 2024-07-29 at 11.01.06 PM.png…]()

**Comment by AgustinBerardi** [comment]
I had the same issue before the last release (version 0.1.0). With the new update (v0.1.1), here is the new error:
<img width="1499" alt="Screenshot 2024-07-29 at 11 01 06 PM" src="https://github.com/user-attachments/assets/3c14f040-b86e-4d07-ab50-333a30fff656">

**Comment by lusingander** [comment]
If you just post a screenshot, it doesn't tell us anything.

- Please clarify the terminal and version you are using, the protocol you are specifying, etc.
- Please provide a repository for reproducing the issue.
  - Does it work with a simple repository?

**Comment by lusingander** [comment]
> I'm not entirely sure what your specific problem is, but https://github.com/lusingander/serie/pull/5 has been merged so that might solve your problem.

The fix for #5 was not properly reflected in the release 😢
https://github.com/lusingander/serie/pull/20

**Comment by lusingander** [comment]
Since there is no additional information, we will close this issue for now.
If necessary, please reopen or create a new issue [with information to reproduce it](https://github.com/lusingander/serie/blob/master/CONTRIBUTING.md#reporting-bugs).
